CloudHealth/helm
# CHE-158 - Consolidate a few external PRs

Documentation links:

    JIRA Ticket(s): [CHE-158](https://cloudhealthtech.atlassian.net/browse/CHE-158)

Proposed changes:

- Add Release.namespace in service account -- Will make sure Pods will run on the same namespace as entered in helm chart install
- Add pod annotations support in deployment
- Remove 'watch' from ClusterRole permissions. Only get and list are needed.
- Updated version to 1.0.1 and added contributors list. All changes to Chart.yaml. It shows up in index.yaml.

Test Plan and Risk Assessment:

How did you test this locally?

 - Use minikube and local collection endpoint. Use the new helm chart to install collector on local minikube and check collection endpoint logs
 - Copy the helm chart (.tgz file of 1.0.1) to install collector on a kubernetes cluster running on Azure Kubernetes Service. Make sure cluster shows up as healthy. Screenshots below.
 
![image](https://user-images.githubusercontent.com/87672958/157353783-4eff7a3f-dda2-473d-992c-57d58d3d7b09.png)
![image](https://user-images.githubusercontent.com/87672958/157353881-dac26e22-b288-448e-8d54-a40bbd852f1a.png)


How will you test this in production?

    Will test helm chart on GCP Kubernetes Service.

Rollback Plan:

    Remove 1.0.1 helm chart (.tgz file) from our repo. This will ensure we will install the previous version.
